PostFreeSurfer/scripts/CreateMyelinMaps_1res.sh

```diff
@@ -126,12 +126,7 @@ for STRING in ${STRINGII} ; do
   Mesh=`echo $STRING | cut -d "@" -f 2`
   ROI=`echo $STRING | cut -d "@" -f 3`
 
-  STRINGIIList="corrThickness@shape"
-  if [ "${T2wPresent}" = "YES" ] ; then
-    STRINGIIList+=" MyelinMap@func SmoothedMyelinMap@func MyelinMap_BC@func SmoothedMyelinMap_BC@func"
-  fi
-
-  for STRINGII in $STRINGIIList ; do
+  for STRINGII in $STRINGIList ; do
```

Comment:
Since we are using `STRINGIList` again, unless we are inside the previous `STRINGI` loop, we should probably use `STRINGI` as the loop variable again, otherwise it looks like a bug.

Reply:
I think this is a typo.  There is no `STRINGIList` (or `STRINGI`) variable.  It should be
`  for STRINGII in $STRINGList ; do`

Reply:
Fixed.